### PR TITLE
Sentieon qualcal

### PR DESCRIPTION
**Pull request by FriederikeHanssen**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

**Comment by FriederikeHanssen** [comment]
@nf-core-bot fix linting

File: modules/nf-core/sentieon/qualcal/tests/nextflow.config [hunk 1]
```diff
@@ -0,0 +1,9 @@
+env {
+    // NOTE This is how nf-core/sarek users will use Sentieon in real world use
```
Comment (SPPearce) [inline modules/nf-core/sentieon/qualcal/tests/nextflow.config:2]: Not just `nf-core/sarek` users, could be other places too.

**Review by SPPearce — APPROVED** [review]

File: modules/nf-core/sentieon/qualcal/tests/nextflow.config (hunk 1 above)
Comment (FriederikeHanssen) [reply modules/nf-core/sentieon/qualcal/tests/nextflow.config:2]: would like to update this in on e sweep across all modules, but can do it right now too

Comment (SPPearce) [reply modules/nf-core/sentieon/qualcal/tests/nextflow.config:2]: Fine, feel free to do it in one go.

**Pull request merged**